tests/test_main.py

```diff
@@ -343,3 +344,17 @@ def test_format_linter_report(errors_linter, errors_mate):
         f"Function 'format_linter_report' should return {errors_mate} "
         f"when 'errors' equals to {errors_linter}"
     )
+
+
+def test_comment_deleted():
+    lines = inspect.getsource(main)
+    assert "# write your code here" not in lines, (
+        "Remove the unnecessary" " comment '# write your code here'"
+    )
```

Comment:
This test `test_comment_deleted` is not necessary. The presence of comments in the code does not affect the functionality of the code. It's better to focus on tests that verify the correctness of the code logic.